paddle/fluid/inference/api/details/zero_copy_tensor.cc

```diff
@@ -74,6 +74,19 @@ T *ZeroCopyTensor::data(PaddlePlace *place, int *size) const {
   return res;
 }
 
+PaddleDType ZeroCopyTensor::type() {
+  EAGER_GET_TENSOR;
+  auto type = tensor->type();
+  if (type == framework::proto::VarType::FP32) {
+    return PaddleDType::FLOAT32;
+  } else if (type == framework::proto::VarType::INT64) {
+    return PaddleDType::INT64;
```

Comment:
We support INT32 as well.

Reply:
Yeah.  At present, int32 will not be used of the VIS model and this branch will then merge to develop.